zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -1482,10 +1484,13 @@ mod tests {
                 assert_eq!(DiversifierIndex::from(tv.diversifier_index), di);
                 assert_eq!(tvua.transparent(), ua.transparent());
                 assert_eq!(tvua.sapling(), ua.sapling());
+                #[cfg(not(feature = "orchard"))]
                 assert_eq!(tv.unified_addr, ua.encode(&Network::MainNetwork));
 
+                // hardcoded with knowledge of what's coming next
+                let ua_request = UnifiedAddressRequest::unsafe_new(false, true, true);
```

Comment:
Should this also be `UA_TRANSPARENT` like before?

Reply:
No; the whole test is under the `transparent-inputs` feature.